src/pages/City/City/CityDocuments.tsx

```diff
@@ -27,6 +27,9 @@ const CityDocuments = () => {
       setLoading(true);
       try {
         const response = await getCityById(+id);
+        const response1 = await cityNameOfApprovedMember(userApi.getActiveUserId());
```

Comment:
Please, provide an appropriate name
This name doesn't meet with conventions

Reply:
Changed this